lib/files.h

```diff
@@ -141,6 +141,18 @@ tool_rc files_save_ESYS_TR(ESYS_CONTEXT *ectx, ESYS_TR handle, const char *path)
  */
 bool files_save_public(TPM2B_PUBLIC *public, const char *path);
 
+
+/**
+ * Serializes a TPMT_PUBLIC to the file path provided.
+ * @param public
```

Comment:
rename the param from public to template. the TPMT_PUBLIC is the public template details. Since we're using public usually for the TPM2B_PUBLIC, this will help prevent confusion when glancing at the code.